panels/nerf_3dview_panels/dataset_panel.py

```diff
@@ -66,6 +70,12 @@ def draw(self, context):
             nerf = NeRFManager.get_nerf_for_obj(nerf_obj)
             
             if nerf.dataset is not None:
+                dataset_path = nerf.dataset.file_path
+                parent_folder = os.path.basename(os.path.dirname(dataset_path))
+                file_name = os.path.basename(dataset_path)
```

Comment:
```suggestion
                parent_folder = Path(dataset_path).parent.name
                file_name = dataset_path.name
```

I generally prefer `Pathlib` to `os` because it has a much nicer syntax and is easier to read.  You can also do stuff like this with pathlib:

```
some_file = Path("root_directory") / "file.txt"
```
Here, the `/` operator turns this into the literal path "root_directory/file.txt" - it's very nice.